SourceCode/support/convert_masterfiles_to_csv.py

```diff
@@ -157,14 +149,15 @@ def find_last_time_masterfile_modified(models, model, scen, dir_masterfiles):
     return master_time_mod > csv_time_mod
 
 def get_remaining_variables(vars_to_convert, out_dir, model, \
-                            var_dict, gamma_options, overwrite_existing):
+                            var_dict, gamma_options, overwrite_existing_csvs):
     """
-    Remove variables from list to convert if two conditions are met:
+    Remove variables from list to convert if either of two conditions are met:
         * Condition 1: csv files already exist or
         * Condition 2: The newest csv file is older than the masterfile
```

Comment:
at first glance I thought list to convert are the variables we are going to update from the master. Am I wrong? If not then  surely we want to remove those variables that are newer than the master? Also if it is 'either' condition to be met, any file that exists will be removed, it doesn't matter if it is newer or older

Reply:
Also there is a comment showing where condition 1 comes in but not 2